### PR TITLE
Initialization script

### PR DESCRIPTION
**Pull request by sjahreis**
Add a script which allows to create a new project without having to clone the project manually.

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
```
Comment (philkarlsson) [inline init.sh:3]: Three variants to check:
- Without parameter --> Interactive mode
- Three parameters --> direct execution mode
- Else --> Show info how to use this script

File: init.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+set +e
+GENERATION_SUCCEEDED=$(./generate.sh "$DEST_FOLDER" "$GROUP_ID" "$ARTIFACT_ID")
+if [[ "$GENERATION_SUCCEEDED" != 0 ]]; then
+  echo "Error during project generation, see the previous logs for more details!"
+fi
+set -e
+
+echo "Removing temporary checkout directory ($CHECKOUT_DIR) ..."
+rm -rf "$CHECKOUT_DIR"
+
```
Comment (philkarlsson) [inline init.sh:57]: Echo the path where the app is now installed including the directory which is generated

**Review by betacore — CHANGES_REQUESTED** [review]
Please also describe how to use this within the README.
bash command to directly use the script without clone and the variants for directly with parameters and the interactive mode.

File: init.sh (hunk 2 above)
Comment (sjahreis) [reply init.sh:57]: Added

File: init.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+if [[ -z $1 && -z $2 && -z $3 && -z $4 ]]; then
```
Comment (philkarlsson) [inline init.sh]: use this instead:
if [ "$#" -ne 1 ]; then

File: init.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  echo "Starting in interactive mode ..."
+  read -p "Enter destination-folder for your new project [./ktor-example]: " -r DEST_FOLDER_INPUT
+  DEST_FOLDER=${DEST_FOLDER_INPUT:-./ktor-example}
+  read -p "Enter group-id of your new project [com.linked-planet]: " -r GROUP_ID_INPUT
+  GROUP_ID=${GROUP_ID_INPUT:-com.linked-planet}
+  read -p "Enter artifact-id of your new project [ktor-example]: " -r ARTIFACT_ID_INPUT
+  ARTIFACT_ID=${ARTIFACT_ID_INPUT:-ktor-example}
+  read -p "Enter branch of ktorbase repo [master]: " -r GIT_BRANCH_INPUT
+  GIT_BRANCH=${GIT_BRANCH_INPUT:-master}
+elif [[ -z $1 || -z $2 || -z $3 || -z $4 ]]; then
```
Comment (philkarlsson) [inline init.sh]: see comment above

**Review by betacore — APPROVED** [review]
LGTM

**Pull request merged**